tests/vmi_lifecycle_test.go

```diff
@@ -1212,8 +1225,19 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 		})
 
-		Context("with non default namespace", func() {
+		Context("with non default namespace", decorators.WgS390x, func() {
 			DescribeTable("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:component]should log libvirt start and stop lifecycle events of the domain", func(alternativeNamespace bool) {
+				arch := util.TranslateBuildArch()
```

Comment:
nit - inline below

Reply:
review comment not applicable after reverting the changes.